ui/src/app/cohort-search/modifier-page/modifier-page.component.ts

```diff
@@ -160,15 +167,16 @@ export class ModifierPageComponent implements OnInit, OnDestroy {
     }
 
   currentMods(vals) {
-    return this.modifiers.map(({name, inputType, modType}) => {
+      this.ngAfterContentChecked();
+      return this.modifiers.map(({name, inputType, modType}) => {
       const {operator, valueA, valueB} = vals[name];
       const between = operator === 'BETWEEN';
       if (!operator || !valueA || (between && !valueB)) {
         return ;
       }
       if (inputType === 'date') {
-          this.dateValueA = moment(valueA, 'MM/DD/YYYY').format('YYYY-MM-DD');
-          this.dateValueB = moment(valueB, 'MM/DD/YYYY').format('YYYY-MM-DD');
+          this.dateValueA = moment(valueA).format('YYYY-MM-DD');
+          this.dateValueB = moment(valueB).format('YYYY-MM-DD');
           const operands = [this.dateValueA];
           if (between) { operands.push(this.dateValueB); }
```

Comment:
should we check that this.dateValueB is set to something instead of the between const? This feels like a subtle bug to me? If so then we can remove the const between.. it will not be needed.